.idea/LabA2020_2.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module org.jetbrains.idea.maven.project.MavenProjectsManager.isMavenModule="true" type="JAVA_MODULE" version="4">
```

Comment:
To fix committing unnecessary files, you can use ".gitignore" file which describes folders and files which are not needed to be committed. You can put "target" and ".idea" folders there.
Note: you need to remove these files from repository manually or via GIT, or via IDE. If you remove them using Finder - GIT will not catch these deletions and not delete them from the index.